src/screens/Notifications/ChooseWalletsForNotificationScreen.tsx

```diff
@@ -64,22 +77,58 @@ export class ChooseWalletsForNotificationScreen extends PureComponent<Props, Sta
     </View>
   );
 
+  onFailure = () => {
+    Alert.alert(this.props.error);
```

Comment:
there is no handling via alert in the design

Reply:
I wait for Joanna response, but how would you handle this error?

Reply:
Confirmed - no alerts. Plus 'VirtualizedLists should never be nested inside plain ScrollViews' Warning fixed